#! /bin/sh /usr/share/dpatch/dpatch-run
## xx.dpatch by  <root@debian>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: xx

@DPATCH@

diff -ur -xdebian -x.svn ../trunk/codec/Makefile ./codec/Makefile
--- ../trunk/codec/Makefile	2007-03-01 16:12:17.000000000 +0000
+++ ./codec/Makefile	2007-11-27 23:14:20.000000000 +0000
@@ -4,10 +4,10 @@
 
 all: j2k_to_image image_to_j2k
 
-j2k_to_image: j2k_to_image.c ../libopenjpeg.a
+j2k_to_image: j2k_to_image.c ../libopenjpeg.so
 	gcc $(CFLAGS) compat/getopt.c convert.c j2k_to_image.c -o j2k_to_image -L.. -lopenjpeg -I ../libopenjpeg/ -lm -ltiff
 
-image_to_j2k: image_to_j2k.c ../libopenjpeg.a
+image_to_j2k: image_to_j2k.c ../libopenjpeg.so
 	gcc $(CFLAGS) compat/getopt.c convert.c image_to_j2k.c -o image_to_j2k -L.. -lopenjpeg -I ../libopenjpeg/ -lm -ltiff
 
 clean:
Only in ./codec: Makefile~
Only in ./codec: Makefile.rej
